common/url_constants.cc

```diff
@@ -13,3 +13,5 @@ const char kRewardsScheme[] = "rewards";
 const char kWidevineMoreInfoURL[] = "https://www.eff.org/issues/drm";
 const char kWidevineTOS[] = "https://policies.google.com/terms";
 const char kRewardsUpholdSupport[] = "https://uphold.com/en/brave/support";
+const char kP3ALearnMoreURL[] = "https://brave.com/P3A";
+const char kP3ASettingsLink[] = "chrome://settings/privacy";
```

Comment:
can this be `brave` or does that still not work?

Reply:
last time I tried it still didn't work

Reply:
that is correct - it didn't work with `brave://` unfortunately